modules/hivemq/gradle.properties

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
Can we move the version into the Gradle file? Should also make it easier for Dependabot.